tests/EntityManager.postgre.test.ts

```diff
@@ -322,6 +322,76 @@ describe('EntityManagerPostgre', () => {
     await expect(orm.em.rollback()).rejects.toThrowError('An open transaction is required for this operation');
   });
 
+  test.only('test nested find with repository', async () => {
+    const mock = mockLogger(orm, ['query', 'query-params']);
+
+    const book = new Book2('My Life on The Wall, part 1');
+
+    const author = new Author2('Bartleby', 'bartelby@writer.org');
+    author.books.add(book);
+
+    await orm.em.persistAndFlush(author);
+
+    orm.em.clear();
+
+    const reqEm = orm.em.fork();
+
+    async function requestCommonService() {
+      const [b] = await reqEm.getRepository(Book2).find(Book2);
```

Comment:
this is not valid, the parameter needs to be query, not class, something like this
```suggestion
      const [b] = await reqEm.getRepository(Book2).find(book);
```

Reply:
in the second test you are doing `em.find(Book2, {})`